examples/objectives_examples.ipynb

```diff
@@ -0,0 +1,384 @@
+   "metadata": {},
+   "source": [
+    "This time, the solver chooses to start with task two to meet its due date precisely. The setup time is taken for granted, as also the due date of task one is met. This leads to 0 tardiness, but a makespan of 12.\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**Info:** Difference between `deadline` and `due_date`\n",
+    "\n",
+    "The `deadline` of a job is strict, leading to infeasibility when it is not met. In contrast, a job's `due_date` is not strict and can be violated. Violation of a `due_date` is taken into account in tardiness-based objectives.\n",
+    "\n",
+    "</div>\n",
+    "\n",
+    "Suppose you are interested in minimizing the weighted sum of the makespan and the total tardiness. In that case, you can set both objectives with different weights, as shown next. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-12-04T19:04:47.684392Z",
+     "start_time": "2024-12-04T19:04:47.655875Z"
+    }
+   },
+   "source": [
+    "model.set_objective(\n",
+    "    weight_makespan=1,\n",
+    "    weight_total_tardiness=2,\n",
+    ")  # all other weights are 0\n",
+    "result = model.solve(display=False)\n",
+    "print(\"Solver status =\", result.status)\n",
+    "print(\"Objective value =\", result.objective)\n",
+    "print(\"Makespan =\", result.best.makespan)\n",
+    "for idx, task in enumerate(result.best.tasks):\n",
+    "    print(f\"Start time Task {idx} =\", task.start)"
+   ],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Solver status = SolveStatus.OPTIMAL\n",
+      "Objective value = 4.0\n",
+      "Makespan = 2\n",
+      "Start time Task 0 = 0\n",
+      "Start time Task 1 = 1\n"
+     ]
+    }
+   ],
+   "execution_count": 4
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, the total tardiness is twice as important as the makespan. As a result, the solver starts with task one again, leading to an objective value of 1 * 2 + 2 * 1 = 4 (`weight_makespan * makespan + weight_total_tardiness * total_tardiness`).  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Different job weights"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In practice, jobs can have different priorities. For example, you would like to meet the due date of a new customer to show your best service. To differentiate between the importance of jobs, you can set job weights as shown in the following single machine example with two jobs. Job one is three times as important as job two in this example. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-12-04T19:04:47.699393Z",
+     "start_time": "2024-12-04T19:04:47.687393Z"
+    }
+   },
+   "source": [
+    "from pyjobshop import Model\n",
```

Comment:
This import statement can be removed because it's already imported at the beginning of the notebook.